Intersect.Server.Framework/Plugins/Helpers/IMapHelper.cs

```diff
@@ -0,0 +1,11 @@
+﻿using Intersect.Server.Framework.Items;
+using Intersect.Server.Framework.Maps;
+
+namespace Intersect.Server.Plugins.Helpers;
+
+public interface IMapHelper
+{
+    event Action<IItemSource, IItem> ItemAdded;
+    IMapInstance GetMapInstanceByDescriptorId(Guid mapId, Guid mapInstanceId);
```

Comment:
Honestly this should be a Try, since this _could_ return null.

Change to `bool TryGetMapInstance(Guid mapId, Guid instanceId, [NotNullWhen(true)] out IMapInstance? instance)`

The implementation should be `returnMapController.TryGetInstanceFromMap(mapId, instanceId, out instance);`